src/components/pages/import.js

```diff
@@ -1,152 +1,149 @@
-import by from 'thenby';
```

Comment:
Shall we rename this file maybe to `batch-imports.js`? I feel it can get confusing between `imports` and `import`, when both contain imports of multiple documents.
I think this page will be under the route `imports/<batchId>`, so it would correspond with the file name.

Or `imports-batch.js`, `create-imports.js` -> `imports-create.js`? And then all 3 files reflect their URL route and are in the file system one after the other?

What do you think?

Reply:
Or `imports.js`, `imports-view.js`, `imports-create.js`? 😬

Reply:
I'm currently following the convention in the codebase. We have for example "docs.js" for all docs and doc.js for one doc. It's a bit confusing, I know @ahelmberger what's your opinion, I for one like Adriana's sugggestion

Reply:
Yeah for docs and doc it makes sense cause it's clear in itself. But for imports not so much, it would make more sense like `import-batches` and `import-batch-view`, `import-batch-create` (👈  which is also a good combination 😂 )

Reply:
The page names mostly follow the URLs (loosely), so as long as they're kinda related, I'm fin with all proposed filenames above.

Reply:
I went for:
import-batches
import-batch-view
create-import-batch (this one follows our other "create-something" pattern in the codebase)
Urls:
import-batches
import-batches/:batchId
import-batches/create?source=abc

I hope that's alright :).